src/DotNetLightning.Core/Crypto/KeyExtensions.fs

```diff
@@ -285,6 +286,7 @@ module KeyExtensions =
                                    : TransactionSignature * PSBT =
             let htlcPrivKey = perCommitmentPoint.DeriveHtlcPrivKey this.HtlcBasepointSecret
             let htlcPubKey = htlcPrivKey.HtlcPubKey()
+            psbt.Settings.CustomBuilderExtensions <- ([new HTLCReceivedExtensions() :> BuilderExtension; new HTLCOfferedExtensions():> BuilderExtension] |> Seq.ofList)
```

Comment:
nits:
- missing a space before `:>`
- please split line in many, it's too long